tools/assert-complete-migrations.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+set -euo pipefail
+
+# This script asserts that each migration file in conf/evolutions
+# - is mentioned in either MIGRATIONS.released.md or MIGRATIONS.unreleased.md (but not both)
+# - has a reversion sibling in conf/evolutions/reversions
+
+EVOLUTIONS_FOLDER="conf/evolutions"
+RELEASED_FILE="MIGRATIONS.released.md"
+UNRELEASED_FILE="MIGRATIONS.unreleased.md"
+
+CONTENT_RELEASED=$(cat "$RELEASED_FILE")
+CONTENT_UNRELEASED=$(cat "$UNRELEASED_FILE")
+
+exit_code=0
+
+for file in "$EVOLUTIONS_FOLDER"/*; do
+    filename=$(basename "$file")
+
+    # Skip the "reversions" directory and initial-schema
+    if [[ "$filename" == "reversions" || "$filename" == "000-initial-schema.sql" ]]; then
+        continue
+    fi
+
+    mentioned_in_x=$([[ "$CONTENT_RELEASED" == *"$filename"* ]] && echo 1 || echo 0)
+    mentioned_in_y=$([[ "$CONTENT_UNRELEASED" == *"$filename"* ]] && echo 1 || echo 0)
+
+    if (( mentioned_in_x == 1 && mentioned_in_y == 1 )); then
+        echo "Conflict: '$filename' is listed in both MIGRATIONS.released.md and MIGRATIONS.unreleased.md."
```

Comment:
```suggestion
        echo "Conflict: '$filename' is listed in both $RELEASED_FILE and $UNRELEASED_FILE."
```
(or something like that? same below)